scripts/collect-snmp-data.php

```diff
@@ -27,8 +28,12 @@
 );
 
 if (isset($options['snmpsim'])) {
-    $snmpsim = new Snmpsim();
-    $snmpsim->run();
+    try {
+        $snmpsim = new Snmpsim();
+        $snmpsim->run();
+    } catch (SnmpsimException $e) {
```

Comment:
This was removed wasn't it?

Reply:
Thought I removed all of them, haven't had a chance to double check.  Thanks.